### ISSUE TITLE
Request to Make the Table of Contents Collapsible in Skin:Citizen

### ISSUE THREAD

**Issue by pppery** [StarCitizenTools/mediawiki-skins-Citizen#622]
## Feature summary (what you would like to be able to do and where):
I want a collapsible table of contents in Skin:Citizen.

## Use case(s) (list the steps that you performed to discover that problem, and describe the actual underlying problem which you want to solve. Do not describe only a solution):
I found the table of contents in Skin:Citizen is not collapsible.

## Benefits (why should this be implemented?):
If it is achieved, then I can read some long text like tables easily.

Refiled from https://phabricator.wikimedia.org/T336629

**Comment by alistair3149** [comment]
Yes it is a feature that I'm looking into but I don't have an ETA for it currently.

**Comment by ZoruaFox** [comment]
> Yes it is a feature that I'm looking into but I don't have an ETA for it currently.是的，这确实是我正在研究的功能，但目前还没有明确的时间表。

I’ve noticed that the latest version of Citizen now uses a separate right-hand sidebar for the table of contents. Should this issue be marked as resolved?

**Comment by alistair3149** [comment]
Yes since the ToC has now become the sidebar, #996 is a more accurate issue for this. Closing this for now.
